Sources/Commands/SwiftTool.swift

```diff
@@ -642,6 +651,7 @@ public class SwiftTool<Options: ToolOptions> {
         buildDelegate.isVerbose = isVerbose
 
         let databasePath = buildPath.appending(component: "build.db").pathString
+        BuildSystem.setSchedulerLaneWidth(width: numThreads)
```

Comment:
You should be able to just do `options.numThreads` to avoid having to plumb it everywhere.

Reply:
`BuildSystem` is a class in llbuildSwift. This line is necessary when using libllbuild instead of the command line frontend.